src/app/src/main/resources/ErrorMessageBundle_fr.properties

```
@@ -12,3 +12,6 @@ unexpectedError=Une erreur inattendue s''est produite. \
 userDoesNotExist=L'utilisateur n'existe pas.
 userExists=L'utilisateur existe déjà.
 unknownInputTimeSeriesType=Type de série temporelle d'entrée inconnu
+noPermissionToAccessResource=Vous nêtes pas autorisé à charger cette ressource à partir dADR. Contactez ladministrateur \
```

Some missing apostrophes here I think
```suggestion
noPermissionToAccessResource=Vous n'êtes pas autorisé à charger cette ressource à partir d'ADR. Contactez l'administrateur \
```